Podcast/SearchPeopleTableViewCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  SearchPeopleTableViewCell.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 3/6/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol SearchPeopleTableViewDelegate {
+    func searchPeopleTableViewCell(cell: SearchPeopleTableViewCell, didPressFollowButtonFor user: User, newValue: Bool)
+}
+
+class SearchPeopleTableViewCell: UITableViewCell {
+    
+    let imageViewPaddingX: CGFloat = 18
+    let imageViewPaddingY: CGFloat = 18
+    let imageViewWidth: CGFloat = 40
+    let imageViewHeight: CGFloat = 40
+    let imageViewLabelPadding: CGFloat = 12
+    let nameLabelPaddingY: CGFloat = 22
+    let nameLabelHeight: CGFloat = 17
+    let detailLabelHeight: CGFloat = 15
+    let followButtonPaddingX: CGFloat = 18
+    let followButtonPaddingY: CGFloat = 21
+    let followButtonHeight: CGFloat = 34
+    let followButtonWidth: CGFloat = 73
+    
+    var profilePictureImageView: UIImageView!
+    var nameLabel: UILabel!
+    var detailLabel: UILabel!
+    var followButton: UIButton!
+    
+    var user: User?
+    var delegate: SearchPeopleTableViewDelegate?
+    
+    var followButtonPressed: Bool = false {
```

Comment:
drew made a fill button that has similar properties, you should probably use that button here